source/packages/services/greengrass2-provisioning/src/templates/templates.models.ts

```diff
@@ -114,6 +116,5 @@ export interface DeploymentPolicy {
     };
 }
 
-export type AbortConfigFailureType = 'FAILED' | 'REJECTED' | 'TIMED_OUT' | 'ALL';
```

Comment:
Am I reading this right that we're now getting this type from their library instead of defining it ourselves? Does the same not apply to the other exported types below? If so, can you open an issue to update them to use the library values instead of defining them in our code?

Reply:
I believe the newer version of the dependent AWS SDK package updated their typing from a general `string` to more strict `enum`, which results in this incompatibility. And you are right, this can apply to other types defined in the code base. I have created https://github.com/aws/aws-connected-device-framework/issues/211 to track this